### PR TITLE
contrib: Extend and add autoyast profiles

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Review by Martchus — CHANGES_REQUESTED** [review]
The PR title doesn't match the commit messages at all.

**Comment by okurz** [comment]
> The PR title doesn't match the commit messages at all.

It does now

File: contrib/ay-openqa-worker-leap-3-nvme.xml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    </drive>
+    <drive>
+      <type t="symbol">CT_DISK</type>
+      <device>/dev/nvme0n1</device>
+      <disklabel>gpt</disklabel>
+      <use>all</use>
+      <partitions t="list">
+        <partition>
+          <size>max</size>
+          <filesystem t="symbol">ext4</filesystem>
```
Comment (mimi1vx) [inline contrib/ay-openqa-worker-leap-3-nvme.xml:120]: iont this reformatted by service to ext2 ?

**Review by mimi1vx — APPROVED** [review]

File: contrib/ay-openqa-worker-leap-3-nvme.xml (hunk 1 above)
Comment (okurz) [reply contrib/ay-openqa-worker-leap-3-nvme.xml:120]: only if we actually use that service which is AFAIK only the case in OSD with salt

Comment (Martchus) [reply contrib/ay-openqa-worker-leap-3-nvme.xml:120]: I think ext4 is a good default. If it gets overridden that's not a problem.

**Review by Martchus — APPROVED** [review]

**Pull request merged**